simplq/src/components/common/Popup/QrCode.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useRef, forwardRef } from 'react';
+import { makeStyles, Modal, useMediaQuery } from '@material-ui/core';
+import { QRCode } from 'react-qrcode-logo';
+import PrintIcon from '@material-ui/icons/Print';
+import { useReactToPrint } from 'react-to-print';
+import StandardButton from '../Button';
+
+const ComponentToPrint = forwardRef(({ style, url }, ref) => {
+  return (
+    <div className={style} ref={ref}>
+      <h2>Scan this QR to begin!</h2>
+      <QRCode value={url} />
+      <p>{`or visit ${url}`}</p>
+    </div>
```

Comment:
In the generated PDF, the QR code is coming on the top. Is it possible to add some margin / center the printed component?

![image](https://user-images.githubusercontent.com/11256376/104340011-40d4cd80-551e-11eb-81b6-d89ad23d430c.png)

Also, let's add as a title to the modal component (and the PDF) the queue name in bigger font.

Reply:
Is this Ok?
![Screenshot from 2021-01-12 22-12-49](https://user-images.githubusercontent.com/44055337/104344748-9495e580-5523-11eb-9466-bc006d211649.png)
![Screenshot from 2021-01-12 22-12-54](https://user-images.githubusercontent.com/44055337/104344764-96f83f80-5523-11eb-91b3-4ba17ff02c26.png)
I will commit changes and push

Reply:
This is perfect!

Reply:
That's good!